code/game/gamemodes/vampire/vampire.dm

```diff
@@ -313,6 +313,7 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 		if(ishuman(owner))
 			var/mob/living/carbon/human/V = owner
 			if(issmall(H) && !H.ckey)
+				to_chat(src, "<span class='notice'><b>Your feeding on [H] reduces your hunger, but you get no usable blood from it.</b></span>")
```

Comment:
The 'your' at the beginning is superfluous but this is a very nifty message to have